tools/wheels/cibw_test_command.sh

```diff
@@ -1,3 +1,11 @@
 set -xe
 
+FREE_THREADED_BUILD="$(python -c"import sysconfig; print(bool(sysconfig.get_config_var('Py_GIL_DISABLED')))")"
+if [[ $FREE_THREADED_BUILD == "True" ]]; then
+    # TODO: delete when importing numpy no longer enables the GIL
```

Comment:
Why is it duplicated rather than moved? These lines of code from here on down are no longer needed in `cibw_before_test.sh`, right?